pages/site.yml

```diff
@@ -0,0 +1,18 @@
+brand: Exadel Smart Library
+description: ESL — A Web Components Library
+
+url: "https://esl-ui.com"
+urlPrefix: ""
```

Comment:
do you want to keep it instead of deleting it because it can be useful in the future?

Reply:
Yep, just left it for potential needs in the future. Basically, for the markdown processor (links resolution part), this might appear to be really reasonable.